x-pack/plugins/alerting/server/authorization/alerting_authorization.ts

```diff
@@ -138,6 +138,14 @@ export class AlertingAuthorization {
     return this.authorization?.mode?.useRbacForRequest(this.request) ?? false;
   }
 
+  public getAuthorizedAlertsIndices(owner: string): string | undefined {
```

Comment:
Do you think this would make more sense in the rule registry alerts client for now? Framework has no concept of owner right now.

Reply:
I actually don't see where this is used so maybe it can be removed?

Reply:
@ymao1 Yes you are correct we can remove this. It is an experimental function I was testing out for a `bulkUpdateAlerts` function which lives in another branch. Thought I took it out of this. Thanks for spotting!